src/options/searchInjection/index.jsx

```diff
@@ -0,0 +1,57 @@
+import React, { Component } from 'react'
+
+import { remoteFunction } from 'src/util/webextensionRPC'
+import { Wrapper } from 'src/common-ui/components'
+
+const fetchSearchInjection = remoteFunction('fetchSearchInjection')
+const toggleSearchInjection = remoteFunction('toggleSearchInjection')
+
+class SearchInjectionContainer extends Component {
+    static propTypes = {}
+
+    constructor(props) {
+        super(props)
+        this.state = {
+            searchInjection: true,
+        }
+        this.handleToggle = this.handleToggle.bind(this)
+    }
+
+    async componentWillMount() {
+        const searchInjection = await fetchSearchInjection()
+        this.setState({
+            searchInjection,
+        })
+    }
+
+    async handleToggle(event) {
+        await toggleSearchInjection()
```

Comment:
why not just have `toggleSearchInjection` return the updated value? Not super important, but saves a remote call

Reply:
@poltak  Ok I will see into it !